### PR TITLE
Revert storage sdk

### PR DESCRIPTION
**Pull request by anubhavitis**

### CODE REVIEW

File: src/interface/middleware/canUpload.js [hunk 1]
```diff
@@ -1,37 +1,33 @@
...
-  const limit = await getStorageUse({ contractAddress, invokerAddress });
-  const totalAllowedStorage = limit.storageLimit + limit.extraStorage;
-
-  return limit.storageUse >= totalAllowedStorage;
-}
-
 async function canUpload(req, res, next) {
   const invokerAddress = req.invokerAddress;
   const contractAddress = req.contractAddress;
-  const namespace = req.namespace;
```
Comment (vijaykrishnavanshi) [inline src/interface/middleware/canUpload.js:19]: In this just removing namespace should be fine imo and keeping the old refactored code as it much cleaner.

**Review by vijaykrishnavanshi — CHANGES_REQUESTED** [review]

File: src/infra/database/models/limit.js [hunk 2]
```diff
@@ -4,33 +4,16 @@ const { Schema } = mongoose;
 
 const _limit = {};
 
-const getDefaultStorageLimit = (contractAddress) => {
```
[the remark this answers is not in the document]
Comment (anubhavitis) [reply src/infra/database/models/limit.js:7]: yep, that's why removed it.

File: src/interface/middleware/canUpload.js (hunk 1 above)
Comment (anubhavitis) [reply src/interface/middleware/canUpload.js:19]: alright. I thought so too, but went to reverting the code itself.

File: src/domain/limit/claimStorage.js [hunk 3]
```diff
@@ -18,7 +18,6 @@ async function claimStorage({ contractAddress, invokerAddress }) {
   });
   await Limit.findOneAndUpdate(
     { contractAddress },
-    { invokerAddress },
```
[the remark this answers is not in the document]
Comment (anubhavitis) [reply src/domain/limit/claimStorage.js:21]: this is syntax error. Missed due to lack of proper testing on lower environments.

**Review by vijaykrishnavanshi — APPROVED** [review]

**Pull request merged**